src/presenters.rs

```diff
@@ -332,80 +337,24 @@ impl Presenter {
         self.current_frame = 0;
     }
 
-    fn clear_swapchain_image(
```

Comment:
I removed this function because it wasn't obvious that this was doing anything useful (because vulkan renderpasses can set their own clear color) but it was causing synchronization issues with race conditions to the framebuffer